gef-ui/templates/includes/footer.njk

```diff
@@ -6,10 +6,12 @@
     <div class="govuk-footer__meta">
       <div class="govuk-footer__meta-item govuk-footer__meta-item--grow">
         <h2>Contact us</h2>
-        <p class="email-wrap">
-          Email: DigitalService.TradeFinance@ukexportfinance.gov.uk<br/>
-          Opening times: Monday to Friday, 9am to 5pm (excluding public holidays)
-        </p>
+        <dl class="contact-details">
+          <dt>Email:</dt>
+          <dd>DigitalService.TradeFinance@ukexportfinance.gov.uk</dd><br>
```

Comment:
```suggestion
          <dd>DigitalService.TradeFinance@ukexportfinance.gov.uk<br></dd>
```

Reply:
![image](https://github.com/UK-Export-Finance/dtfs2/assets/116292912/5816d3ca-27a9-48b9-b339-8f2a03f2bc9b)

Dev tools flags that `<br>` is not a valid child of `<dl>`. https://dequeuniversity.com/rules/axe/4.4/definition-list suggests this a problem because of how screen readers announce elements. This is the change suggested by https://stackoverflow.com/questions/23687838/how-to-add-a-line-break-br-in-dl-in-html and removes the warning without changing the appearance of the page.

Reply:
I checked example and maybe our case is different.
I removed `<br>` and added styling (new line) using css.